curated_transformers/util/serde.py

```diff
@@ -18,10 +34,52 @@
 ]
 
 
+class ModelCheckpointType(Enum):
+    """
+    Types of model checkpoints supported by Curated Transformers.
+    """
+
+    #: PyTorch checkpoint.
+    PYTORCH = 1
+
+    #: Hugging Face `Safetensors <https://github.com/huggingface/safetensors>`_ checkpoint.
+    SAFETENSORS = 2
```

Comment:
They call them `SafeTensors` in the Rust source, so maybe a good excuse to add an underscore to the name here?

Reply:
Done.